src/hooks/useValidateAppVersion.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { Linking, Platform } from 'react-native';
+import Config from 'react-native-config';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+enum RemoteFields {
+  MINIMUM_VERSION = '',
```

Comment:
Enum holds a set of named constants, it is required that you explicitly add here the value tight to the `MINIMUM_VERSION` const.

``` suggestion
  MINIMUM_VERSION = 'minimum_version',
```

If you leave it as an empty string the RemoteConfig library will try to fetch an invalid config.

Reply:
Done